commands/sdcli_go_integration

```diff
@@ -27,7 +27,7 @@ INTTESTS=$(go test -v -tags=integration -cover -coverpkg="${PKGS}" -coverprofile
 FAIL="$(echo "${INTTESTS}" | grep 'FAIL')"
 if [[ ${FAIL} != "" ]]; then
     echo "A setup or compilation failure occurred."
```

Comment:
Why does this statement assume a failing integration test is due to a setup or compilation failure? it could be an actual bug or broken test.

Reply:
It's a badly worded message.  In verbose terms, it could/should say `A setup, compilation, or test failure occurred.`

At the moment, if one were to have a compilation failure in an `integration_test.go` file, vscode won't tell the developer due to the `// +build integration` comment at the top, so developers run the integration tests via sdcli, where the compilation failure would be first detected in the lazy vs-code-using developer's workflow.